src/CSI.IBTA.DataLayer/Data/UnitOfWork.cs

```diff
@@ -11,18 +11,14 @@ public class UnitOfWork : IUnitOfWork, IDisposable
         private readonly CsiHealthcare2024Context _context;
         private readonly ILogger _logger;
 
-        public IUserRepository Users { get; private set; }
-        public IAccountRepository Accounts { get; private set; }
-        public IEmployerRepository Employers { get; private set; }
+        public IGenericRepository<User> Accounts { get; private set; }
 
         public UnitOfWork(CsiHealthcare2024Context context, ILoggerFactory loggerFactory)
         {
             _context = context;
             _logger = loggerFactory.CreateLogger("logs");
 
-            Users = new UserRepository(context, _logger);
-            Accounts = new AccountRepository(context, _logger);
-            Employers = new EmployerRepository(context, _logger);
+            Accounts = new GenericRepository<User>(context, _logger);
```

Comment:
```suggestion
            Users = new GenericRepository<User>(context, _logger);
```
I dunno what is the Db table name, but lets make naming consistent. DB name should match with DTO and Repo name to avoid confusion